deploy/utils.sh

```diff
@@ -157,6 +155,152 @@ function deploy_env {
+    -e "s#{{ CONJUR_APPLIANCE_URL }}#${CONJUR_APPLIANCE_URL:-"https://conjur-follower.${CONJUR_NAMESPACE_NAME}.svc.cluster.local/api"}#g" \
+    -e "s#{{ CONJUR_AUTHN_URL }}#${CONJUR_AUTHN_URL:-"https://conjur-follower.${CONJUR_NAMESPACE_NAME}.svc.cluster.local/api/authn-k8s/${AUTHENTICATOR_ID}"}#g" \
+    -e "s#{{ CONJUR_AUTHN_LOGIN }}# ${CONJUR_AUTHN_LOGIN:-"host/conjur/authn-k8s/${AUTHENTICATOR_ID}/apps/${APP_NAMESPACE_NAME}/*/*"}#g"  \
+    -e "s#{{ SECRETS_PROVIDER_SSL_CONFIG_MAP }}# ${SECRETS_PROVIDER_SSL_CONFIG_MAP:-"secrets-provider-ssl-config-map"}#g" \
+    -e "s#{{ IMAGE_PULL_POLICY }}# ${IMAGE_PULL_POLICY:-"IfNotPresent"}#g" \
+    -e "s#{{ IMAGE }}# ${IMAGE:-"$image_path/secrets-provider"}#g" \
+    -e "s#{{ TAG }}# ${TAG:-"latest"}#g" \
+    -e "s#{{ LABELS }}# ${LABELS:-"app: test-helm"}#g" \
+    -e "s#{{ DEBUG }}# ${DEBUG:-"false"}#g" \
+    -e "s#{{ RETRY_COUNT_LIMIT }}# ${RETRY_COUNT_LIMIT:-"3"}#g" \
+    -e "s#{{ RETRY_INTERVAL_SEC }}# ${RETRY_INTERVAL_SEC:-"30"}#g" \
+    "$helm_path/helm/secrets-provider/ci/test-values-template.yaml" > "$helm_path/helm/secrets-provider/ci/${id}test-values.yaml"
+}
+
+fill_helm_chart_no_override_defaults() {
+  helm_path="."
+  if [ "${DEV}" = "false" ]; then
+    helm_path=".."
+  fi
+
+  image_path="${APP_NAMESPACE_NAME}"
+  if [[ "${PLATFORM}" = "openshift" && "${DEV}" = "false" ]]; then
+    # Image path needs to point to internal registry path to access image
+    image_path="docker-registry.default.svc:5000/${APP_NAMESPACE_NAME}"
+  elif [[ "${PLATFORM}" = "kubernetes" && "${DEV}" = "false" ]]; then
+    image_path="${DOCKER_REGISTRY_PATH}/${APP_NAMESPACE_NAME}"
+  fi
+
+  sed -e "s#{{ K8S_SECRETS }}#${K8S_SECRETS:-"test-k8s-secret"}#g" \
+    -e "s#{{ DEBUG }}# ${DEBUG:-"false"}#g" \
+    -e "s#{{ LABELS }}# ${LABELS:-"app: test-helm"}#g" \
+    -e "s#{{ CONJUR_ACCOUNT }}#${CONJUR_ACCOUNT:-"cucumber"}#g" \
+    -e "s#{{ CONJUR_APPLIANCE_URL }}#${CONJUR_APPLIANCE_URL:-"https://conjur-follower.${CONJUR_NAMESPACE_NAME}.svc.cluster.local/api"}#g" \
+    -e "s#{{ CONJUR_AUTHN_URL }}#${CONJUR_AUTHN_URL:-"https://conjur-follower.${CONJUR_NAMESPACE_NAME}.svc.cluster.local/api/authn-k8s/${AUTHENTICATOR_ID}"}#g" \
+    -e "s#{{ CONJUR_AUTHN_LOGIN }}# ${CONJUR_AUTHN_LOGIN:-"host/conjur/authn-k8s/${AUTHENTICATOR_ID}/apps/${APP_NAMESPACE_NAME}/*/*"}#g"  \
+    -e "s#{{ IMAGE_PULL_POLICY }}# ${IMAGE_PULL_POLICY:-"IfNotPresent"}#g" \
+    -e "s#{{ IMAGE }}# ${IMAGE:-"$image_path/secrets-provider-sigal-4"}#g" \
+    -e "s#{{ TAG }}# ${TAG:-"latest"}#g" \
+    "../helm/secrets-provider/ci/take-defaults-test-values-template.yaml" > "../helm/secrets-provider/ci/take-defaults-test-values.yaml"
+}
+
+deploy_chart() {
+  pushd ../
+    fill_helm_chart
+    helm install -f "helm/secrets-provider/ci/test-values.yaml" \
+      secrets-provider ./helm/secrets-provider \
+      --set-file environment.conjur.sslCertificate.value="conjur.pem"
+  popd
+}
+
+configure_env_path() {
```

Comment:
why is this called `configure_env_path` if it sets the CONFIG_DIR variable? can it have a proper name?

Reply:
Updating to `set_config_directory_path`